src/reducers/tokenPair.ts

```diff
@@ -1,16 +1,36 @@
 import { handleActions } from 'redux-actions'
 
 import { selectTokenAndCloseOverlay, selectTokenPair, setSellTokenAmount, swapTokensInAPair } from 'actions'
-import { TokenPair, Balance } from 'types'
-import { TokenItemProps } from 'components/TokenItem'
+import { TokenPair, DefaultTokenObject, TokenMod } from 'types'
 
-export default handleActions<TokenPair & { sellAmount: Balance }, TokenItemProps & TokenPair>(
+// TODO: fill in state with dispatch in or around getDefaultTokens
+const initialState: TokenPair = {
+  sell: {
+    name: "ETHER",
+    symbol: "ETH",
+    address: "0x254dffcd3277c0b1660f6d42efbb754edababc2b",
```

Comment:
@Velenir from `truffle network` - this is TokenGNO and your TokenGNO = RDN. Just change these to:
```
  sell: {
    name: "ETHER",
    symbol: "ETH",
    address: "0xcfeb869f69431e42cdb54a4f4f105c19c080a601",
    decimals: 18,
  },
  buy: {
    name: "GNOSIS",
    symbol: "GNO",
    address: "0x254dffcd3277c0b1660f6d42efbb754edababc2b",
    decimals: 18,
  },
```

Reply:
good catch